src/app/assets/stylesheets/functions.scss

```diff
@@ -4,3 +4,8 @@
         max-height: inherit;
     }
 }
+
+.CodeMirror {
+    border: 1px solid #eee;
```

Comment:
Question: do we need to set the border property? and the height property is it needed also?